askama_derive/src/generator.rs

```diff
@@ -727,6 +727,52 @@ impl<'a> Generator<'a> {
                 args.len()
             )));
         }
+        // Handling both named and unnamed arguments requires to be careful of the arguments order.
```

Comment:
I think I'd prefer a more restrictive way of combining named and unnamed arguments, which is that we only allow named arguments after positional arguments.

Reply:
Please update this comment to be in line with the updated logic.